### PR TITLE
EXP: Investigate Windows failure on Travis CI

### PR DESCRIPTION
**Pull request by pllim**
### Description

Windows job on Travis CI suddenly cannot find `python.exe`. Larry saw same problem over at `photutils`. Not sure what had changed.

p.s. My job is queued. Looks like this gonna take a while.

### CODE REVIEW

**Comment by pllim** [comment]
I think maybe `tox` 3.20.1 (released 2020-10-09) broke stuff?

**Comment by mhvk** [comment]
Same failure for `pyerfa` - checked travis windows pages but nothing obvious.

Any changes on the (mini)conda side?

**Comment by pllim** [comment]
Miniconda is already pinned over at `ci-helpers`. 🤷

File: .travis.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ env:
         # that tox passes through to the {posargs} indicator in tox.ini.
         # The latter can be used for example to pass arguments to pytest.
         - TOXENV='test'
-        - TOXARGS='-v'
+        - TOXARGS='-rv'
```
Comment (pllim) [inline .travis.yml:51]: @astrofrog and @Cadair , is this where I am supposed to apply `-r`? If it is, didn't help.

File: tox.ini [hunk 2]
```diff
@@ -8,6 +8,7 @@ requires =
     setuptools >= 30.3.0
     pip >= 19.3.1
     tox-pypi-filter >= 0.12
+    virtualenv == 20.0.33
```
Comment (pllim) [inline tox.ini:11]: Nope, can't do this either. `could not satisfy requires MissingDependency(<Requirement('virtualenv==20.0.33')>)`

**Pull request closed without merging**